src/canmatrix/copy.py

```diff
@@ -21,10 +21,14 @@
 # DAMAGE.
 
 from __future__ import absolute_import
+import logging
 
 from .canmatrix import *
 from copy import deepcopy
 
+logger = logging.getLogger(__name__)
```

Comment:
Maybe don't preemptively create this if unused.  Also noted below.

Reply:
Similar to xlsx, the copy module used wrong logger instance comming from `from .canmatrix import *`.